.github/workflows/tests.yml

```diff
@@ -124,6 +124,63 @@ jobs:
           steps: ${{ toJson(steps) }}
         if: failure()
 
+  test-postgres15:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_DB: alerta
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.10']
```

Comment:
Do we need to still support Python 3.7 for this?

Reply:
Some of the docker builds use Python 3.7